### PR TITLE
display category into template

### PR DESCRIPTION
**Pull request by ayeniblessing101**
Hi Dima.... Having issues displaying the data with range. Have no idea why Its not working

### CODE REVIEW

File: internal/handlers/category.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<h1>Category Page</h1>
+{{range .Categories}}
```
Comment (dkotik) [inline internal/handlers/category.html]: try {{ range . }} here

**Review by dkotik — APPROVED** [review]
See the hint below:

File: internal/handlers/category.html (hunk 1 above)
Comment (ayeniblessing101) [reply internal/handlers/category.html]: It worked.... Thanks

Comment (ayeniblessing101) [reply internal/handlers/category.html]: What next?

Comment (ayeniblessing101) [reply internal/handlers/category.html]: The data passed is .

Comment (dkotik) [reply internal/handlers/category.html]: Do you understand why it worked? Next, let's re-write the HttpHandler for go fiber (make sure you are using /v2). Here is the documentation:

https://github.com/gofiber/fiber

Comment (ayeniblessing101) [reply internal/handlers/category.html]: What I understand is that every data passed to a template can be accessed using {{ . }}

**Pull request merged**